assets/src/components/amp-support/validated-urls.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ListItems } from '../list-items';
+
+/**
+ * To render validated urls information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for validated urls data.
+ */
+export function ValidatedUrls( { data: validatedUrls } ) {
```

Comment:
Check out my comments around the `Plugins` component since they apply here as well.